crates/bevy_transform/src/components/transform.rs

```diff
@@ -26,14 +26,22 @@ use std::ops::Mul;
 /// This system runs in stage [`CoreStage::PostUpdate`](crate::CoreStage::PostUpdate). If you
 /// update the [`Transform`] of an entity in this stage or after, you will notice a 1 frame lag
 /// before the [`GlobalTransform`] is updated.
+///
+/// Examples: [`transform`](https://github.com/bevyengine/bevy/blob/latest/examples/transforms/transform.rs), [`global_vs_local_translation.rs`](https://github.com/bevyengine/bevy/blob/latest/examples/transforms/global_vs_local_translation.rs)
```

Comment:
We should settle on either including or excluding the `.rs` extension, but avoid mixing styles.